### DISCUSSION TITLE
Add .cursorrules file

### DISCUSSION THREAD

**Discussion by NickAnthony** [t3-oss/create-t3-turbo#1272] [Ideas]
### Describe the feature you'd like to request

I'd like a `.cursorrules` file that outlines:

1. Structure of the repo
2. Libaries/tooling used
3. Style preferences

### Describe the solution you'd like to see

Just a .cursorrules file.  Should be straightforward.

### Additional information

Make the cursor agents much better at writing code.

**Comment by juliusmarminge** [comment]
hmm not sure

Related: https://x.com/thomaspaulmann/status/1867117681486201041
